region/wreckedship/main.json

```diff
@@ -2884,6 +3092,26 @@
                     "If Phantoon is alive, 'Base' can be used instead.",
                     "Note that this strat is useless in vanilla, as the item only spawns if Phantoon has been killed."
                   ]
+                },
+                {
+                  "name": "WS East Supers Robot Clip Run-In-Place",
+                  "notable": true,
+                  "requires": [
+                    {"not": "f_DefeatedPhantoon"},
+                    "canRiskPermanentLossOfAccess",
```

Comment:
```suggestion
```

Reply:
This may show up in the spoiler log funny because this strat doesn't have a resource cost but you can do the conveyor run strat without getting hit too so I should put a tech indicating that possibilty.